modules/calc/src/test/java/com/opengamma/strata/calc/config/pricing/PricingRuleTest.java

```diff
@@ -30,36 +31,32 @@
 @Test
 public class PricingRuleTest {
 
-  private static final Measure MEASURE1 = Measure.of("1");
-  private static final Measure MEASURE2 = Measure.of("2");
-  private static final Measure MEASURE3 = Measure.of("3");
-
   private static final FunctionGroup<TestTrade1> GROUP =
       DefaultFunctionGroup.builder(TestTrade1.class)
           .name("GroupName")
-          .addFunction(MEASURE1, FunctionConfig.of(TestFunction1.class))
-          .addFunction(MEASURE2, FunctionConfig.of(TestFunction2.class))
+          .addFunction(Measures.PRESENT_VALUE, FunctionConfig.of(TestFunction1.class))
```

Comment:
They haven't been lost, I've just replaced a locally defined constant with an externally defined constant.

Previously it was possible to define a local measure with an arbitrary name. Now they have to match the built in set of measures. In which case it makes sense to use the standard measures rather then defining local aliases.